modules/contact-form/grunion-contact-form.php

```diff
@@ -3434,6 +3466,23 @@ function render_checkbox_field( $id, $label, $value, $class, $required, $require
 		return $field;
 	}
 
+	/**
+	 * Render the consent field.
+	 *
+	 * @param string $id field id.
+	 * @param string $class html classes (can be set by the admin).
+	 * @param string $consent_type can be implicit or explicit.
+	 * @param string $consent_message message that needs to be displayed to the visitor.
+	 */
+	private function render_consent_field( $id, $class, $consent_type, $consent_message ) {
+		$field  = "<label class='grunion-field-label consent consent-" . $consent_type . "'>";
+		$field .= "\t\t<input type='checkbox' " . checked( 'implicit' === $consent_type, true, false ) . " name='" . esc_attr( $id ) . "' value='" . esc_attr__( 'Yes', 'jetpack' ) . "' " . $class . ' ' . ( 'implicit' === $consent_type ? "style='display:none;'" : '' ) . "/> \n";
```

Comment:
Would it be an option to not render the checkbox when one picks the implicit option, to avoid issues for folks using screen readers, for example?

Reply:
I see 2 options:
1) Adjust the existing PHP code that captures the form entries, today it needs an input field to get the existing value when it's submitted. I can make an exception to the consent field.
2) add aria-hidden="true" it skip the screen readers.

@jeherve I believe option 2 is cleaner. let me know your thoughts?

Reply:
I like option 2 as well 👍